icu4c/source/common/rbbitblb.h

```diff
@@ -15,6 +15,9 @@
 #define RBBITBLB_H
 
 #include "unicode/utypes.h"
+
+#if !UCONFIG_NO_BREAK_ITERATION
```

Comment:
It's unclear why this started needing an #if now. It's an internal header that is only used by the RBBI implementation. With UCONFIG_NO_BREAK_ITERATION set, nothing should include it.

Reply:
When compiling this header stand-alone compiler error is:
./common/rbbitblb.h:62:36: error: unknown type name 'IntPair'

'IntPair' is defined in common/rbbirb.h and is filtered out when UCONFIG_NO_BREAK_ITERATION is set.